src/main/java/Duke.java

```diff
@@ -1,10 +1,165 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    //CONSTANTS
+    public static final String SEPARATOR = "____________________________________________________________\n";
+
+    //METHODS
+    public static void printGreeting() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n"
+                + logo
+                + SEPARATOR
+                + " Hello! I'm Duke\n"
+                + " What can I do for you?\n"
+                + SEPARATOR);
```

Comment:
Good that you broke the println into multiple lines, made it very clear how the final string will look like